src/client/components/Google_login_button.components/GoogleLoginButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './GoogleLoginButton.css';
+import GoogleLogo from '../../assets/images/Google_logo.png';
+
+const GoogleLoginButton = () => {
+  return (
+    <>
+      <button className="google_login_button" type="submit">
+        <img
+          className="google_logo_img"
+          alt="Google_logo_for_login"
```

Comment:
the alt attribute is to show to client in case the image is not available, you could name it "Google logo" instead.

Reply:
Yes, true. Me and my namings :D